tests/PHPUnit/Resolver/FormTypeResolverTest.php

```diff
@@ -69,5 +69,6 @@ public function provideEmailKeyAndExpectedForm(): \Generator
         yield '[Plus] Return request resolution changed' => ['sylius_plus_return_request_resolution_changed', \Synolia\SyliusMailTesterPlugin\Form\Type\SyliusPlusReturnRequestType::class];
         yield '[Plus] Return request repaired items sent' => ['sylius_plus_return_request_repaired_items_sent', \Synolia\SyliusMailTesterPlugin\Form\Type\SyliusPlusReturnRequestType::class];
         yield '[Plus] Loyalti purchase coupon' => ['sylius_plus_loyalty_purchase_coupon', \Synolia\SyliusMailTesterPlugin\Form\Type\SyliusPlusLoyaltyPurchaseCouponType::class];
+        yield 'Invoice generated' => ['invoice_generated', \Synolia\SyliusMailTesterPlugin\Form\Type\Plugin\InvoicingPlugin\InvoiceType::class];
```

Comment:
I think you need to add the InvoicePlugin in dev-dependencies if yoyu whant this test pass. Else, you can remove this line :slightly_smiling_face: